src/docs/cookbook-payments/customize-pricing/index.md

```diff
@@ -466,409 +270,198 @@ The input can be styled by adding the following style definitions to
-  ```
+### Capture add-ons value for the booking breakdown
 
-Now the `estimatedTotalPrice` function can be renamed as
-_"estimatedRentalPrice"_:
+In the `BookingDatesForm` `handleOnChange` function, resolve if the
+cleaning fee is selected and pass the information to the
+`onFetchTransactionLineItems` function:
 
 ```diff
-- const estimatedTotalPrice = (unitPrice, unitCount) => {
-+ const estimatedRentalPrice = (unitPrice, unitCount) => {
-    const numericPrice = convertMoneyToNumber(unitPrice);
-    const numericTotalPrice = new Decimal(numericPrice).times(unitCount).toNumber();
-    return new Money(
-      convertUnitToSubUnit(numericTotalPrice, unitDivisor(unitPrice.currency)),
-      unitPrice.currency
-    );
-  };
-```
-
-Then create a new `estimatedTotalPrice` function that includes
-`cleaningFee`:
+ handleOnChange(formValues) {
+   const { startDate, endDate } =
+     formValues.values && formValues.values.bookingDates ? formValues.values.bookingDates : {};
+   const listingId = this.props.listingId;
+   const isOwnListing = this.props.isOwnListing;
+
++  const addOns =
++    formValues.values && formValues.values.addOns ? { addOns: formValues.values.addOns } : {};
+
+   if (startDate && endDate && !this.props.fetchLineItemsInProgress) {
+     this.props.onFetchTransactionLineItems({
+-      bookingData: { startDate, endDate },
++      bookingData: { startDate, endDate, ...addOns },
+       listingId,
+       isOwnListing,
+     });
+   }
+ }
+```
+
+There, now the listing page booking panel indicates if a listing has a
+cleaning fee configured and it can be selected when making a booking.
+Next, let's move to the server side and update our pricing parameters.
+
+## 3. Update pricing logic
+
+Flex pricing uses
+[privileged transitions](/background/privileged-transitions/) to ensure
+flexible pricing models while keeping control of the pricing logic in a
+secure environment. Transitioning requests of privileged transitions are
+made from server-side. Thus we'll need to update the pricing logic in
+the `/server/api-util/lineItems.js` file:
 
-```js
-const estimatedTotalPrice = (unitPrice, unitCount, cleaningFee) => {
-  const numericPrice = convertMoneyToNumber(unitPrice);
-  const cleaningFeePrice = cleaningFee
-    ? convertMoneyToNumber(cleaningFee)
-    : null;
-  const numericTotalPrice = cleaningFeePrice
-    ? new Decimal(numericPrice)
-        .times(unitCount)
-        .plus(cleaningFeePrice)
-        .toNumber()
-    : new Decimal(numericPrice).times(unitCount).toNumber();
-  return new Money(
-    convertUnitToSubUnit(
-      numericTotalPrice,
-      unitDivisor(unitPrice.currency)
-    ),
-    unitPrice.currency
-  );
-};
+```shell
+└── server
+    └── api-util
+        └── lineItems.js
 ```
 
-And the function invocation respectively:
+_NOTE: When editing server side files, remember to run the application
+with `yarn run dev-server` in order for the changes to take effect._
```

Comment:
This is not true anymore. `yarn run dev` starts also a dev server, but code is not hotloaded automatically. So, the command needs a restart.

The server side pricing wont work if the server is not up and running.

Reply:
I built a cleaning fee yesterday for screenshot purposes. Running `yarn run dev` is enough but also you don't need to restart to get the server side changes live. Saving edits is enough.

Reply:
Yeah, just confirmed this, my bad. Indeed, as @ovan said, just `yarn run dev`.